config/dev.exs

```diff
@@ -75,5 +75,5 @@ if System.get_env("CLOUDEX_API_KEY") == nil do
 end
 
 config :code_corps,
-  github_oauth_client_id: System.get_env("GITHUB_OAUTH_CLIENT_ID"),
-  github_oauth_client_secret: System.get_env("GITHUB_OAUTH_CLIENT_SECRET")
+  github_app_client_id: System.get_env("GITHUB_APP_CLIENT_ID"),
+  github_app_client_secret: System.get_env("GITHUB_APP_CLIENT_SECRET")
```

Comment:
I think maybe this could all just be folded into one config rather than duplicated across them.